### PR TITLE
made carrie

### PR DESCRIPTION
**Pull request by carriescott**
made carrie

### CODE REVIEW

**Review by pete85 — CHANGES_REQUESTED** [review]
Change something

**Review by bayramcura — CHANGES_REQUESTED** [review]
Please change code section in carrie.md file to add text DONE....

File: friends-and-colleagues/carrie.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- and others  
+
+
+
+## Frameworks and Technologies that I know:
+
+- Many of the most important frameworks, plus a number you've never heard of
+
+
+Contact information available upon request
```
Comment (bayramcura) [inline friends-and-colleagues/carrie.md:25]: This content should include DONE XYZ...

**Review by pete85 — APPROVED** [review]

**Review by mugshepherd — APPROVED** [review]
this looks great

**Review by Chandrima6 — APPROVED** [review]
SDF

**Comment by mugshepherd** [comment]
SDF?

**Comment by mugshepherd** [comment]
@bayramcura ?

**Review by bayramcura — APPROVED** [review]
Great job

**Pull request merged**